### PR TITLE
docs(operate): add section: monitor operation status

### LINKED ISSUE #3654: Document how to monitor operation status in Operate user guide
### Description

It will be possible in the Operate UI to see operation status for each process instance that is related to an operation. This needs to be documented in the Operate user guide.

### TODO

- [x] add new section to [Operate user guide](https://docs.camunda.io/docs/next/components/operate/operate-introduction/): "Monitor operation status"
- [x] add instructions with screenshots to that section guiding the user through the user story described in the [issue](https://github.com/camunda/operate/issues/6356) 
- [x] replace section [Monitor the migration progress ](https://docs.camunda.io/docs/next/components/operate/userguide/process-instance-migration/#monitor-the-migration-progress) by a link to "Monitor operation status" section
- [x] add a link to "Monitor operation status" section to [Selections and operations](https://docs.camunda.io/docs/next/components/operate/userguide/selections-operations/)

### Related issues
- https://github.com/camunda/operate/issues/6356

### PR DESCRIPTION
**Pull request by pedesen**
## Description

- add new section to Operate user guide: monitor operation status
- add links to that section in the following sections:
  - Selections and operations
  - Process instance migration
  - Process instance modification
  - Process instance batch modification

closes #3654 

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [x] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by christinaausley** [comment]
@pedesen Would you like this to go out with the alpha next week?

**Comment by pedesen** [comment]
@christinaausley would be nice, if that's possible.

File: docs/components/operate/userguide/process-instance-modification.md [hunk 1]
```diff
@@ -141,3 +141,7 @@ Some elements do not support specific modifications:
 - **Add token**/**Move tokens to** modifications are currently not possible for elements with multiple running scopes.
 
 ![not-supported-flow-nodes](../../../images/operate/modifications/not-supported-flow-nodes.png)
+
+## Monitor the modification status
```
Comment (christinaausley) [inline docs/components/operate/userguide/process-instance-modification.md:145]: Do you mean "monitor the operation status" here?

File: docs/components/operate/userguide/monitor-operation-status.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+description: "Monitor and investigate the status of any operation"
+---
+
+Operate offers a powerful way to monitor and investigate the status of any operation you started. This includes canceling and retrying process instances, process instance migration and modification.
+
+First go to the processes view and expand the operations panel:
+
+![operate-view-process-filters](./img/monitor-operation-state/operations-panel.png)
+
+Select an operation from the list by clicking on the operation id. Notice that the operation if is set as a filter:
```
Comment (christinaausley) [inline docs/components/operate/userguide/monitor-operation-status.md]: Should we add a note in the case the user doesn't have any operations and sees "No operations have been created yet."?

File: docs/components/operate/userguide/process-instance-modification.md (hunk 1 above)
Comment (pedesen) [reply docs/components/operate/userguide/process-instance-modification.md]: Modification is a specific type of operation. I think it makes sense to call it "modification operation status". I rephrased it.

File: docs/components/operate/userguide/process-instance-migration.md [hunk 3]
```diff
@@ -54,14 +54,6 @@ Now, you can see a preview of how flow nodes are mapped and how many process ins
 
 ![operate-view-process-filters](./img/process-instance-migration/summary.png)
 
-## Monitor the migration progress
+## Monitor the migration status
```
[the remark this answers is not in the document]
Comment (pedesen) [reply docs/components/operate/userguide/process-instance-migration.md]: Same here, but "migration operation status".

File: docs/components/operate/userguide/monitor-operation-status.md (hunk 2 above)
Comment (pedesen) [reply docs/components/operate/userguide/monitor-operation-status.md]: Good idea, I added it.

**Review by christinaausley — APPROVED** [review]

**Pull request merged**